lib/sarif/yarn_audit_sarif.rb

```diff
@@ -0,0 +1,76 @@
+require 'salus/bugsnag'
+module Sarif
+  class YarnAuditSarif < BaseSarif
+    include Salus::SalusBugsnag
+    YARN_URI = 'https://classic.yarnpkg.com/en/docs/cli/audit/'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = YARN_URI
+      parse_scan_report!
+    end
+
+    def parse_scan_report!
+      @logs = @scan_report.to_h.fetch(:logs).split("\n\n")
+    rescue KeyError => e
+      bugsnag_notify(e.message)
+      @logs = []
+    end
+
+    def parse_issue(issue)
```

Comment:
This could the be shortened to
```
    def parse_issue(issue)
      id = issue["ID"]
      return nil if @issues.include?(id)

      @issues.add(id)
      {
        id: id,
        name: issue['Title'],
        level: issue['Severity'].upcase,
        details: "Title: #{issue['Title']}\nPackage: #{issue['Package']}\nPatched in: #{issue['Patched in']}"\
        "\nDependency of:#{issue['Dependency of']} \nSeverity: #{h['Severity']}",
        uri: "yarn.lock",
        help_url: issue['More info']
      }
    end```